.github/workflows/release-provider.yml

```diff
@@ -13,10 +13,15 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
       - name: Setup Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v2.1.3
         with:
+<<<<<<< HEAD
           go-version: 1.13
       - uses: actions/cache@v1
+=======
+          go-version: 1.15
+      - uses: actions/cache@v2.1.3
+>>>>>>> 813e646... Merge pull request #92 from mercari/upgrade-go-version
```

Comment:
```suggestion
          go-version: 1.15
      - uses: actions/cache@v2.1.3
```

Reply:
Oh, I missed that 😢  I will fix right away 🏃  Thank you 🙇

Reply:
I edited the commit of `Merge pull request #92 from mercari/upgrade-go-version` which contained that unresolved merge conflicts. And I ported(cherry-picked) the three PRs you suggested. And then I force pushed. 

PTAL 🙇